vm/src/stdlib/collections.rs

```diff
@@ -471,6 +471,23 @@ mod _collections {
             })
         }
 
+        #[pymethod(magic)]
+        fn imul(zelf: PyRef<Self>, value: isize, vm: &VirtualMachine) -> PyResult<PyRef<Self>> {
+            let mut new_deque: VecDeque<PyObjectRef> = {
```

Comment:
No reason to repeat ourselves here though, right? Either `imul` calls `mul` or `mul` calls `imul`, both can work. CPython has `mul` call into `imul` after doing a copy. For now, this can just call into `mul` and then swap the deques. We can time it at some later point to see if the opposite yields any better results (though I'm guessing the difference would be negligible).